CHANGELOG.md

```diff
@@ -39,7 +39,9 @@ A brief description of the categories of changes:
   containing ">" sign
 
 ### Added
-* Nothing yet
+* (pypi) Support env markers in requirements files. Note, that this means that
+  if your requirements files contain env markers, the Python interpreter will
+  have to be downloaded to re-evaluate the bzlmod extension.
```

Comment:
"re-evaluate the bzlmod extension" ? bzlmod extensions don't get re-evaluated? They have a well-defined dependency ordering. Do you mean the underlying repos during the repo phase, because they'll trigger a restart?

Reply:
Clarified this. My mind was thinking about re-evaluation of the extension if any underlying dependencies change so it was a slip of a finger here.